rs/bitcoin/kyt/src/providers.rs

```diff
@@ -3,15 +3,78 @@ use ic_btc_interface::Txid;
 use ic_cdk::api::management_canister::http_request::{
     CanisterHttpRequestArgument, HttpHeader, HttpMethod, TransformContext, TransformFunc,
 };
+use std::fmt;
 
-pub fn create_request(
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+enum ProviderId {
```

Comment:
Great news!!! After asking my friend at Blockstream a few times (and him asking the tech guys internally), blockstream.info now supports IPv6!  :tada: 

It has less restrictive rate limiting than mempool.space, so I'd rank it higher (but that doesn't matter if we employ a round-robin strategy).